src/cljs/main/broadfcui/page/workspace/monitor/common.cljs

```diff
@@ -5,16 +5,22 @@
    [broadfcui.common.icons :as icons]
    [broadfcui.common.style :as style]
    [broadfcui.common.table.style :as table-style]
-   [broadfcui.config :as config]
    [broadfcui.utils :as utils]
-   ))
+   [broadfcui.components.foundation-dropdown :as dropdown]))
 
 
 (defn render-date [date]
   (if date
     (str (common/format-date date) " (" (duration/fuzzy-time-from-now-ms (.parse js/Date date) true) ")")
     [:span {:style {:fontStyle "italic"}} "Pending..."]))
 
+(defn render-cost [cost]
```

Comment:
Moved from `submission_details.cljs` - now shared between it and `workflow_details.cljs`